bibformat/format_elements/bfe_INSPIRE_enhanced_marcxml.py

```diff
@@ -121,6 +122,14 @@ def get_matched_id(subfields):
     return None
 
 
+def salvage_deleted_record_from_history(recid):
+    return create_record(decompress(run_sql("SELECT marcxml FROM hstRECORD WHERE id_bibrec=%s ORDER BY job_date DESC LIMIT 1", (recid, ))[0][0]))[0]
```

Comment:
just for clarification, what is the purpose of retrieving the most recent version from history instead of the current record which is simply marked `980:DELETED`. When I look at a random deleted rec
```
In [3]: x=search_unit('DELETED', f='980', m='a')

In [4]: len(x)
Out[4]: 36033

In [5]: x[-300]
Out[5]: 1494279

In [7]: oldrec = get_record(1494279)

In [11]: hstrec = create_record(decompress(run_sql("SELECT marcxml FROM hstRECORD WHERE id_bibrec=%s ORDER BY job_date DESC LIMI
    ...: T 1", (1494279, ))[0][0]))[0]

In [16]: sorted(hstrec.keys())
Out[16]: 
['001',
 '005',
 '024',
 '035',
 '100',
 '245',
 '260',
 '300',
 '500',
 '520',
 '540',
 '541',
 '650',
 '773',
 '856',
 '909',
 '970',
 '980',
 '999']

In [17]: sorted(oldrec.keys())
Out[17]: 
['001',
 '005',
 '024',
 '035',
 '100',
 '245',
 '260',
 '300',
 '500',
 '520',
 '540',
 '541',
 '650',
 '773',
 '856',
 '909',
 '970',
 '980',
 '999']
```

they appear to have the same keys. I didn't compare any closer, though.